packages/frontend/.env.prod.public

```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_STORAGE_ORIGIN=https://kids-storage.twreporter.org
 NEXT_PUBLIC_GQL_ENDPOINT=https://kids-api.twreporter.org/api/graphql
+NEXT_PUBLIC_IS_PRODUCTION=true
```

Comment:
Suggestion:
Could we rename it to `NEXT_PUBLIC_RELEASE_ENV`, and its value could be `dev`, `staging` or `prod`?

I think we might need to distinguish which environment the codes run on in the future.

Reply:
okok